blocks/core/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.xsl

```diff
@@ -1,15 +1,17 @@
 <xsl:template name="ff_container-dashboard-section">
     <xsl:param name="data" />
-    <section class="ff_conatiner-dashboard-section {$data/item/@modifiers}">
-        <header class="ff_container-dashboard-section__header">
+    <section class="ff_container-dashboard-section">
+        <header class="ff_container-dashboard-section__header ff_container-dashboard-section__header--{$data/item/@modifiers}">
             <h2 class="ff_container-dashboard-section__title"><xsl:value-of select="$data/item/@title"/></h2>
         </header>
         <div>
             <xsl:attribute name="class">
                 <xsl:text>ff_container-dashboard-section__main</xsl:text>
                 <xsl:if test="$data/item/@show-footer = 'no'"> ff_container-dashboard-section__main--no-footer</xsl:if>
             </xsl:attribute>
-            <p>Modules Go here</p>
+            <xsl:for-each select="$data//module">
```

Comment:
I think it makes more sense for this to be a single node named something like `content` rather than a list of `module`s as it'll always be a single node being passed in to the container.